components/Landing.tsx

```diff
@@ -1,17 +1,63 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
```

Comment:
Is this a client-side rendering component? If it is, we need to add "use client" at the top line to enable Client Component Rendering (CCR).

Reply:
thanks for the reminder, I overlooked adding 'use client' because I didn't encounter any issues during dev.